resolution of issue #265

dexink opened the pull request:
resolves issue #265 per specifications described by @hoganwr

dexink committed:
resolution of issue #265

resolves issue #265 per specifications described by @hoganwr

hoganwr commented on src/ontology/omrse-edit.properties:
We don't check in the ****-edit.properties file. However, I see it's not been added to the .gitignore. I'll do that, and delete this.

hoganwr committed:
Update .gitignore

added src/ontology/omrse-edit.properties

We don't check in these files, they're just a bunch of blank jdbc* properties

hoganwr committed:
Delete src/ontology/omrse-edit.properties

We don't check these in
